Remove claims from dispatcher

GMKrieger opened the pull request:
#40 

Removed claims logic and auxiliary functions and utilities.

Renamed `dispatcher` to `eth-input-reader`.

GMKrieger committed:
refactor(offchain): remove claims from dispatcher

renan061 reviewed (changes requested):
Minor comments!

The CI is broken, probably because of some name that still refers to "dispatcher".

GMKrieger commented:
> Minor comments!
> 
> The CI is broken, probably because of some name that still refers to "dispatcher".

I pushed the fix right before you commented, let's see if it works

GMKrieger commented:
Ok, so the check scopes is failing because the command we use to check is this

`git log --pretty=format:"%s" $(git merge-base HEAD origin/main)..HEAD`

Since this command compares to the main and we're currently branching on next, it's getting previous commits that are not from this branch that includes one with dispatcher as a scope, therefore causing this error.

There's no fix to be done as of right now, just putting this here so anyone that reviews doesn't get confused.

gligneul reviewed (changes requested):
Please don't merge this PR to `next/main` until the claimer is ready. Otherwise we won't be able to release a rollups version.

GMKrieger committed:
refactor(input-reader): update state-fold use

gligneul commented on offchain/eth-input-reader/src/config.rs:
```diff
@@ -19,12 +19,10 @@ use types::deployment_files::{
-#[command(about = "Configuration for rollups dispatcher")]
-pub struct DispatcherEnvCLIConfig {
+#[command(about = "Configuration for rollups eth-input-reader")]
+pub struct EthInputReaderEnvCLIConfig {
```
We still have the `TxEnvCLIConfig`  but the eth-input-reader doesn't send transactions. This seems wrong.

marcelstanley replied:
@GMKrieger FYI

torives committed:
feat: add files to support manual tests

As cartesi-rollups-node gets built and new services are added, it becomes
increasingly cumbersome to set up an environment to test it. The added docker-compose file is meant to simplify the process of manual testing for the
time being

gligneul commented:
https://github.com/cartesi/rollups-node/issues/40#issuecomment-1815299765
